### PR TITLE
Feat/modify res

### PR DESCRIPTION
**Pull request by ritachien**
modify structures and remove unnecessary data in response.

In commits before, response includes unnecessary data. For example...... If a user sign up for a new account, it would return user's account data in old commits. It was a useless and unsafe response content. So I replace the original data with an action success message.

### CODE REVIEW

**Comment by ritachien** [comment]
Add try/catch to addTweet and deleteTweet in this commit

File: controllers/user-controller.js [hunk 1]
```diff
@@ -89,25 +78,16 @@ const userController = {
         })
```
Comment (rubylo718) [inline controllers/user-controller.js:83]: Please check the user stories. 這邊應該是檢查account 與 email不可重複(not account or name)，且若account或email重複，要不同的提示（參考user story有描述錯誤提示文案）

**Review by rubylo718 — COMMENTED** [review]
Please check signup.

File: controllers/user-controller.js (hunk 1 above)
Comment (ritachien) [reply controllers/user-controller.js:83]: 收到! 已處理。

**Review by rubylo718 — APPROVED** [review]
checked ok.

**Pull request merged**